cmake/Config.cmake.in

```diff
@@ -3,6 +3,8 @@
 find_package(OpenCV REQUIRED)
 find_package(ogles_gpgpu CONFIG REQUIRED) # review OpenGL dependencies
 find_package(xgboost CONFIG REQUIRED)
+find_package(acf CONFIG REQUIRED)
+find_package(Boost CONFIG REQUIRED system filesystem)
```

Comment:
Isn't boost a conditional dependency of the library?  Eos?

Reply:
I found usage in Drishti sources here:
* https://github.com/elucideye/drishti/blob/f23764390aeedc5df5e49d956fe41fe50c381a69/src/lib/drishti/face/FaceDetectorFactoryJson.cpp#L18

Reply:
I see.  I'd like to use boost for various features in the application and test layers, but avoid the associated overhead for the library.  There was a fair amount of work involved in removing the boost serialization, so it's a shame to have this one instance.

I was looking for a small portable path manipulation package that would handle platform specific nuances of path (de)construction:

https://github.com/elucideye/drishti/issues/602

I'll take another look.